src/components/MARCTable/MARCTable.js

```diff
@@ -212,10 +198,11 @@ export const MARCTable = ({
       className={css.tableContainer}
     >
       <MARCTableHeader
-        columns={columns}
+        columns={visibleColumns || columns}
```

Comment:
I would suggest moving `columns` to defaultProps and deal only with one variable/prop for that.
```suggestion
        columns={columns}
```